ci/setup_ci_environment.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -e
+apt-get update 
+apt-get install --no-install-recommends --no-install-suggests -y \
+                build-essential \
+                cmake \
+                pkg-config \
+                git \
+                ca-certificates \
+                curl \
+                automake \
+                autogen \
+                autoconf \
+                libtool \
+                gnupg2 \
+                ssh \
+                wget \
+                libgtest-dev
+
+# Follows these instructions for setting up gtest
+# https://www.eriksmistad.no/getting-started-with-google-test-on-ubuntu/
```

Comment:
Webpage says "Note that this package only install source files. You have to compile the code yourself to create the necessary library files." but `dpkg -L libgtest-dev` says

```
/usr/lib/x86_64-linux-gnu/libgtest.a
/usr/lib/x86_64-linux-gnu/libgtest_main.a
```

Reply:
I tried it first without that extra step and cmake couldn't find libgtest

Reply:
Odd. Idk what's going on here so let's keep it as-is for now. :)